src/components/CommitGraph.tsx

```diff
@@ -35,21 +35,39 @@ const CommitGraph = () => {
           'rgba(75, 192, 192, 0.6)',
           'rgba(153, 102, 255, 0.6)',
         ],
+        borderColor: 'transparent',
       },
     ],
   };
 
+  const chartOptions = {
+    plugins: {
+      legend: {
+        labels: {
+          color: '#edf2f7',
+        },
+      },
+    },
+  };
+
   if (ContributorQuery.isLoading) {
-    return <div>Loading...</div>;
+    return (
```

Comment:
Noticed you are doing a few skeletons. You can maybe extract them out and make a shared skeleton component

Reply:
How would that work?

Reply:
Please have a look at it again

Reply:
So what I meant was you can have a component called `<SharedSkeleton />` for example where your loading code can go. I.E 
`<div className="flex h-full w-full flex-col items-center rounded-2xl bg-gray-900 p-4">
        <div className="skeleton h-full w-full bg-gray-800"></div>
</div>` can go in there. Then you have a reusable loading component